android/app/src/main/java/me/kisoft/covid19/HealthWatcher/VitalSignsProcess.java

```diff
@@ -133,11 +163,6 @@ public void onResume() {
         super.onResume();
 
         wakeLock.acquire();
-
-        camera = Camera.open();
-
-        camera.setDisplayOrientation(90);
```

Comment:
Did you delete these? because with this branch the camera doesnt open so maybe here is the issue.

Reply:
I tried adding them but it still didnt work

Reply:
I added them in the onRequestPermissionsResult